### ISSUE TITLE
FeatureGroups

### ISSUE THREAD

**Issue by iandeherdt** [tombatossals/angular-leaflet-directive#281]
Hello,

I can't find a way to create multiple layers and add them as features to the map. Could it be that this functionality is still missing?

I did the following in a test project using default leaflet;

   var leidingenLayer = new L.FeatureGroup(leidingenGEOJSON);
    var hydrantenLayer = new L.FeatureGroup(hydrantenGEOJSON);
    var glasbollenLayer = new L.LayerGroup(markers);
    var hondenloopZoneLayer = new L.LayerGroup(circles);
    
    var features = {
        "glasbollen": glasbollenLayer,
        "hondenloopzones": hondenloopZoneLayer,
        "hydranten": hydrantenLayer,
        "leidingen": leidingenLayer
    };
   
   
    map.addLayer(glasbollenLayer);
    map.addLayer(hondenloopZoneLayer);
    map.addLayer(hydrantenLayer);
    map.addLayer(leidingenLayer);
    var control = new L.Control.Layers(features);
    map.addControl(control);

Basically i want multiple GEOJSON layers that can be turned on and off via javascript and the layer selector.

Thanks in advance,

Ian

**Comment by tombatossals** [comment]
Hi, you can add multiple GeoJON layers now with the nested GeoJSON functionality:

http://tombatossals.github.io/angular-leaflet-directive/examples/0000-viewer.html#/basic/geojson-nested-example
